CMakeLists.txt

```diff
@@ -127,13 +127,13 @@ ELSE()
   IF (NOT KOKKOSKERNELS_HAS_TRILINOS AND NOT KOKKOSKERNELS_HAS_PARENT)
     # This is a standalone build
     FIND_PACKAGE(Kokkos REQUIRED)
-    IF((${Kokkos_VERSION} VERSION_EQUAL "4.1.00") OR (${Kokkos_VERSION} VERSION_GREATER_EQUAL "4.2.00"))
+    IF((${Kokkos_VERSION} VERSION_EQUAL "4.2.00") OR (${Kokkos_VERSION} VERSION_GREATER_EQUAL "4.3.00"))
```

Comment:
Can we simplify this to `IF(${Kokkos_VERSION} VERSION_GREATER_EQUAL "4.2.00")`?
Should make the maintenance just a tiny bit easier and if people do not use the latest patch release that's not really a problem.

Reply:
👍 Good call, I was on autopilot when I made the update, I like the cleanup